playground/backend/internal/code_processing/code_processing.go

```diff
@@ -63,11 +66,10 @@ func Process(ctx context.Context, cacheService cache.Cache, lc *fs_tool.LifeCycl
 		return
 	}
 	executor := executorBuilder.Build()
-
 	// Validate
 	logger.Infof("%s: Validate() ...\n", pipelineId)
 	validateFunc := executor.Validate()
-	go validateFunc(successChannel, errorChannel)
+	go validateFunc(successChannel, errorChannel, &validationResults)
```

Comment:
The map is already a reference type. Why do we need to use `&validationResults` but not `validationResults`?